Setup/Product.wxs

```diff
@@ -15,6 +15,7 @@
     <Property Id="WIXUI_INSTALLDIR" Value="INSTALLDIR" />
     <Property Id="LAUNCH" Value="0" />
     <Property Id="TELEMETRY_ENABLED" Value="1" Secure="yes" />
+    <Property Id="MSIRESTARTMANAGERCONTROL" Value="Disable"/>
```

Comment:
As shell extension affects Explorer,  I believe restart is necessary if shell extension is/was installed.
The code has not changed in some years though, but that is hard to determine.